src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -436,7 +436,7 @@ public Task StartLoadingRemindersAsync(CancellationToken cancellationToken)
                         cancellationToken.ThrowIfCancellationRequested();
                         actor.MarkedForDeletion = true;
 
-                        // Remove actor state(and reminders) first and then unregister reminders as RemoveActorState can throw
+                        // Removes actor state (and reminders) first and then unregister reminders as RemoveActorState can throw
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.